tests/ebpf_xdp.h

```diff
@@ -12,8 +12,8 @@
 
 #include <stdbool.h>
 #include <stdio.h>
-typedef signed char s8; 
-typedef unsigned char u8; 
+typedef signed char s8;
```

Comment:
I wonder whether this file is still necessary - maybe we can use just the corresponding ebpf kernel headers.